guides/20240819_history_embedding.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "Embedding and Searching Browser History with ChromaDB"
+description: "A step-by-step guide to embedding and searching your browser history using ChromaDB using Azure or a local environment."
+date: 2024-08-19
+author: "Jacob Gaffke"
+---
+
+# Embedding and Searching Browser History with ChromaDB
+
+## Introduction
+
+[Embedding](/definitions/d/embedding) your browser history using ChromaDB allows you to transform vast amounts of web activity data into structured, searchable formats. By converting URLs and their associated metadata into numerical representations known as [embedding](/definitions/d/embedding)s, you can efficiently search, filter, and analyze your browsing history. This guide walks you through the process of setting up a development environment, [embedding](/definitions/d/embedding) your browser history, and conducting advanced searches using ChromaDB, whether you’re leveraging a local model or the power of Azure’s cloud-based [embedding](/definitions/d/embedding) services.
+
+This guide is designed for users who want to gain deeper insights into their browsing habits or need a more powerful way to sift through large amounts of history data. From setting up the necessary tools to executing detailed search queries with various filters, you'll gain the knowledge to fully utilize ChromaDB’s capabilities. Whether you’re a developer looking to integrate this functionality into a larger project or simply someone who wants to better manage their digital footprint, these step-by-step instructions will help you achieve your goals.
```

Comment:
target is someone trying to learn embeddings and create their first project, outcome is a project itself